public/js/actions/AtomActions/deleteAtom.js

```diff
@@ -31,11 +22,13 @@ function errorDeletingAtom(error) {
 export function deleteAtom(atom) {
   return dispatch => {
     dispatch(requestAtomDelete(atom));
-    return AtomsApi.DeleteAtom(atom)
+
+    return AtomsApi.deleteAtom(atom)
         .then(res => res.json())
-        .then(atom => {
-          dispatch(receiveAtomDelete(atom));
-        })
-        .catch(error => dispatch(errorDeletingAtom(error)));
+        .then(() =>
+          window.location.href = window.location.origin + '/'
```

Comment:
Any reason for this over using the history from react router?

Reply:
Ah yes sorry, should've left a comment.

The only reason is laziness, `BrowserHistory.push` didn't work when I tried it locally. I suspect it's the same reason why the back button is broken at the moment.

Reply:
@shaundillon and I have discussed IRL and I will raise a separate PR to fix it